frontend/src/app/course/courseCard.jsx

```diff
@@ -0,0 +1,20 @@
+// @flow
+import * as React from 'react';
+import Card from 'react-bootstrap/Card';
+
+import './styles/courseCard.scss';
+import type { Course } from './types';
+
+const CourseCard = (course: Course): React.Node => {
```

Comment:
Create a special props for this component even though it receives only a cours. This component may be modified later with stuff.